recipes/otwrapy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.6" %}
+
+package:
+  name: otwrapy
+  version: {{ version }}
+
+source:
+  fn: otwrapy-{{ version }}.tar.gz
+  url: https://github.com/felipeam86/otwrapy/archive/{{ version }}.tar.gz
+  sha256: f7fb1c4a39a55b96bbe5c91c718b06b063223a2d2fb04442df3197fce47f50c4
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+    - numpy
+    - openturns
```

Comment:
Just to confirm, we aren't linking to these? There is no source code we are building. Also we are stuck with having these at build time as it imports `otwrapy` in `setup.py`?